diagram/diagram.go

```diff
@@ -59,12 +59,13 @@ func (d diagram) Create(result *database.Result) error {
 	}
 
 	var constraints []ErdConstraintData
+	relationshipLabelMap := BuildRelationshipLabelMap(d.config)
```

Comment:
Building the map is `O(n)` and we then get (essentially) `O(1)` lookup times when finding a constraint later vs doing a `O(n)` linear search each time